include/igl/remesh_along_isoline.cpp

```diff
@@ -71,6 +71,7 @@ template <
   }
 
   // Loop over each face
+  std::unordered_map<int, std::unordered_map<int, int>> edgeToBirthVert;
```

Comment:
Even though unordered_map is a C++11 feature you need to `#include <unordered_map>` especially on gcc 4.8 which is used by default in Ubuntu 14.04 (travis-ci).

Reply:
Thank you! I added #include<unordered_map> line.